policyengine_us/variables/gov/states/la/tax/credits/number_of_eligible_children.py

```diff
@@ -0,0 +1,25 @@
+from policyengine_us.model_api import *
+
+
+class number_of_eligible_children(Variable):
+    value_type = int
+    entity = TaxUnit
+    label = "Louisiana non-refundable Child Expense Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.revenue.louisiana.gov/IndividualIncomeTax/SchoolReadinessTaxCredit"
+    defined_for = StateCode.LA
+
```

Comment:
we need to check whether each child in the tax unit is below or above 6 years old. The credit does not cap the number of children but their age